src/data/queries/repositoryQuery.ts

```
@@ -1,9 +1,38 @@
 import { gql } from '@apollo/client'
 import { Repository, RepositoryMetadata } from 'src/data/types'
 import apolloClient from 'src/utils/apolloClient/apolloClient'
+import fetchConditionalCache from 'src/utils/fetchConditionalCache'
```

ESLint

[error] 'fetchConditionalCache' is defined but never used.

(no-unused-vars)